content/500-support/100-help-articles/400 - nextjs-prisma-client-dev-practices.mdx

```diff
@@ -12,7 +12,7 @@ Lots of users have come across this warning while working with [Next.js](https:/
 warn(prisma-client) Already 10 Prisma Clients are actively running
 ```
 
-There's a related [discussion](https://github.com/prisma/prisma/discussions/4399) and [issue](https://github.com/prisma/prisma/issues/5103) for the same. The reason is that in development due to hot reloading, a new instance of `PrismaClient` is created which makes a lot of connections to the database which is not ideal. This leads to database connections getting exhausted because each `PrismaClient` instance holds its own connection pool.
+There's a related [discussion](https://github.com/prisma/prisma/discussions/4399) and [issue](https://github.com/prisma/prisma/issues/5103) for the same. The reason is that in development due to hot reloading, a new instance of `PrismaClient` is created due to `next dev` clearing NodeJS cache which makes a lot of connections to the database in which is not ideal. This leads to database connections getting exhausted because each `PrismaClient` instance holds its own connection pool.
```

Comment:
> The reason is that in development due to hot reloading, a new instance of `PrismaClient` is created due to `next dev` clearing NodeJS cache which makes a lot of connections to the database in which is not ideal.

- this sentence feels long and complicated, can you maybe simplify it? potentially breaking it up in two sentences?
- NodeJS [should be spelled Node.js](https://www.prisma.io/docs/about/style-guide#capitalize-and-spell-out-proper-nouns)